pom.xml

```diff
@@ -44,6 +44,22 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.teiid</groupId>
+      <artifactId>teiid</artifactId>
+      <version>14.0.0</version>
+      <classifier>jdbc</classifier>
+    </dependency>
+    <dependency>
+      <groupId>org.teiid</groupId>
+      <artifactId>teiid-common-core</artifactId>
+      <version>14.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.teiid</groupId>
+      <artifactId>teiid-api</artifactId>
+      <version>14.0.0</version>
+    </dependency>
```

Comment:
These are not needed when above dependency is used, however only if you are strictly using it as JDBC driver